package.json

```diff
@@ -64,12 +68,27 @@
     "typescript": "^4.5.3"
   },
   "dependencies": {
+    "@swc/helpers": "^0.4.14",
     "hoist-non-react-statics": "^3.3.2",
     "launchdarkly-js-client-sdk": "^3.1.1",
     "lodash.camelcase": "^4.3.0"
   },
   "peerDependencies": {
     "react": "^16.6.3 || ^17.0.0 || ^18.0.0",
     "react-dom": "^16.8.4 || ^17.0.0 || ^18.0.0"
+  },
+  "targets": {
+    "main": {
+      "context": "browser",
+      "outputFormat": "commonjs",
+      "isLibrary": true,
+      "optimize": true,
+      "engines": {
+        "browsers": [
+          "> 0.5%, last 2 versions, not dead",
+          "ie >= 11"
+        ]
+      }
+    }
```

Comment:
Thank you for submitting this. Can we please request this to be moved to a [.parcelrc](https://parceljs.org/features/plugins/)?

Reply:
Ah ok, no worries you are right. Thanks for looking into this. I'll approve this and we'll cut a release soon.